packages/next/src/client/components/unresolved-thenable.ts

```diff
@@ -3,4 +3,4 @@
  */
 export const unresolvedThenable = {
   then: () => {},
-}
+} as PromiseLike<void>
```

Comment:
We simplified the types for `Usable` to just accept `PromiseLike` instead of thenables: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69567. I'm not too concerned about this breakage since you could just do `export const unresolvedThenable = Promise.resolve()`, no?